examples/mobile/ScheduleApp/create-event-off.html

```diff
@@ -87,7 +87,7 @@
 			<a class="ui-btn app-event-cancel" data-style="flat" href="#main">
 				Cancel
 			</a>
-			<a class="ui-btn app-event-done" data-style="flat" href="#">
+			<a class="ui-btn app-event-done" data-style="flat" href="#" data-type="off">
```

Comment:
data-* attributes are handled by TAU framework. I'd not mix them application data. How about just "type" atrribute?

Reply:
Data attributes are common HTML attributes. Custom ones without a prefix are possible only for Custom Elements https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes